app/deed/view/routes.py

```diff
@@ -0,0 +1,61 @@
+import requests
+from app.deed.view import views
+from app.domain.borrower import Borrower
+from app.domain.lender import Lender
+from app.domain.land_property import LandProperty
+from flask import redirect
+
+
+def register_routes(blueprint):
+    deed_api_base_host = 'http://localhost:5050'
+
+    @blueprint.route('/deed/view')
+    def view_deed():
+        # handle non existing deed
+        # return redirect('/deed/not_found', 303)
+        borrowers = get_borrowers(get_borrowers_json())
+        lender = get_lender(get_lender_json())
+        land_property = get_land_property(get_property_json())
+        return views.View(borrowers, lender, land_property).render()
+
+    def get_borrowers(borrowers):
+        return [borrower_from_dict(item) for item in borrowers]
```

Comment:
Nice use of the functional style. Just one language feature you might like to know: 

You can define functions inside other functions, scoping them so they can't be used outside of the function they were declared in. So for example your `borrower_from_dict` (which I assume is only used in `get_borrowers`) can be defined inside `get_borrowers` so it can only be used there:

```
def get_borrowers(borrowers):
    def borrower_from_dict(borrower):
        return Borrower(borrower.get('borrower_forename'),
            borrower.get('borrower_surname'),
            borrower.get('borrower_middle'),
            borrower.get('borrower_addr_1'),
            borrower.get('borrower_addr_2'),
            borrower.get('borrower_city'),
            borrower.get('borrower_postcode'))

    return [borrower_from_dict(item) for item in borrowers]
```

It's most useful for it's scoping ability, so it keeps your external interface clean.